etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/sftp/FTPUser.java

```diff
@@ -70,27 +70,7 @@ public boolean promptYesNo(String s) {
 
             //TODO: Implement a common location of the trust file
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1135)